dotnet/src/SemanticKernel/Planning/Plan.cs

```diff
@@ -584,58 +641,6 @@ private ContextVariables GetNextStepVariables(ContextVariables variables, Plan s
         return stepVariables;
     }
 
-    private async Task<SKContext> InstrumentedInvokeAsync(
```

Comment:
The purpose of this method was to move all telemetry-related logic to separate place to avoid mixing telemetry and business-related logic. It looks like this improvement is reverted now. Is there any reason for that?

Reply:
I moved it both in Plan and SKFunction to different method called `InvokeAsyncInstrumented` as you did before.